Lib/fontTools/scaleUpem.py

```diff
@@ -143,14 +143,14 @@ def visit(visitor, varData):
     import sys
 
     if len(sys.argv) != 3:
-        print("usage: scale-upem.py font new-upem")
+        print("usage: fonttools scaleUpem font new-upem")
```

Comment:
maybe fontTools.ttLib.scaleUpem would be a better place than at the top package level, it feels like a special tool that works with the ttLib api more than its own separate "lib"

Reply:
I'll move there if you like. I figured other tools/modules like like subset, merge, fontBuilder are also ttLib-specific but in toplevel.